### DISCUSSION TITLE
2.0.2-compat Release?

### DISCUSSION THREAD

**Discussion by calmyournerves** [tailwindlabs/tailwindcss#3354] [General]
Hey there!

Any plans to release a compat version of release 2.0.2? As it includes a performance improvement (#2820), that would be greatly appreciated.

Thanks!

**Comment by adamwathan** [comment]
Hey! It's already available, we published it at the same time as the PostCSS 8 version:

https://www.npmjs.com/package/@tailwindcss/postcss7-compat

**Reply by calmyournerves** [reply]
Ah, I missed that is has been moved to a separate package. Sorry and thanks!
